components/concept/engine/src/main/java/mozilla/components/concept/engine/prompt/CreditCard.kt

```diff
@@ -25,4 +25,24 @@ data class CreditCard(
     val expiryMonth: String,
     val expiryYear: String,
     val cardType: String
-) : Parcelable
+) : Parcelable {
+    val obfuscatedCardNumber: String
```

Comment:
We have a test for this https://searchfox.org/mozilla-mobile/source/android-components/components/concept/engine/src/test/java/mozilla/components/concept/engine/prompt/CreditCardTest.kt#13. Can you add a check that to confirm that we output the right value?